### DISCUSSION TITLE
Drive formatted with type 2 protection

### DISCUSSION THREAD

**Discussion by QBANIN** [openzfs/zfs#12839] [Q&A]
Hi,

I have 4x ST6000NM0034 formatted with type 2 protection (info from SMART) connected to LSI9211 HBA controller. Is it fine to use them like this with ZFS or should I reformat to remove protection?

> === START OF INFORMATION SECTION ===
> Vendor:               IBM-ESXS
> Product:              ST6000NM0034   X
> Revision:             BC6A
> Compliance:           SPC-4
> User Capacity:        6 001 175 126 016 bytes [6,00 TB]
> Logical block size:   512 bytes
> Physical block size:  4096 bytes
> Formatted with type 2 protection
> LU is fully provisioned
> Rotation Rate:        7200 rpm
> Form Factor:          3.5 inches
> Device type:          disk
> Transport protocol:   SAS (SPL-3)
> 
> 
>

**Comment by zeigerpuppy** [comment]
good discussion of this here:

http://talesinit.blogspot.com/2015/11/formatted-with-type-2-protection-huh.html

In summary, yes, and you can use the command:

```bash
sg_format --format --fmtpinfo=0 /dev/sd<X>
```

where ` <X>` should be replaced with the device (e.g. `sda`)

This can take quite a while, but worth it considering how cheap these drives can be 2nd hand!  (I am formatting 16x 10TB ex-Netapp Ultrastars as we speak...)
